docs/core/building/distribution-packaging.md

```diff
@@ -0,0 +1,63 @@
+* `dotnet-sdk`: the latest SDK. **dependencies**: the latest `dotnet-sdk-[major].[minor]`.
+* `dotnet-sdk-[major].[minor]`: the SDK with the specified version. The version specified is the highest included version of included shared frameworks, so that users can easily relate an SDK to a shared framework. **dependencies**: `dotnet-host`, one or more `dotnet-runtime-[major].[minor]` (one of those is used by the SDK code itself, the others are here for users to build and run against).
+* `dotnet-host`: the latest host.
+
+#### Preview versions
+
+Package maintainers may decide to include preview versions of the shared framework and SDK. Those should never be included in the unversioned `dotnet-sdk` package, but may be released as versioned packages with an additional preview marker appended to the major and minor version sections of the name. For example, there may be a `dotnet-sdk-2.0-preview-final` package.
+
+### Optional additional packages
+
+Some maintainers may choose to provide additional packages such as:
+
+* `dotnet-lts`: the latest "Long-Term Support" version of the shared framework. [LTS and Current release trains](~/docs/core/versions/lts-current.md) correspond to different cadences at which .NET Core gets released. Users have a choice of adopting one or the other train, based on how often they are willing to update. This is a concept that is also tied to support levels, so it may or may not make sense depending on the distro being considered.
+
+## Disk layout
+
+When installing .NET Core packages, the relative placement of their target destinations on disk matter.
+The `dotnet.exe` host should be placed next to `sdk` and `shared` folders that contain the versioned contents of the `dotnet-sdk` SDK packages, and `dotnet-runtime` shared framework packages.
+
+The disk layout of files and directories inside the packages is versioned. This means that updating to the latest `dotnet-runtime` installs the new version side by side with previous versions, reducing the possibility of breaking existing applications by updating the package. Package updates should not remove previous versions.
+
+## Update policies
+
+When an `update` is performed, the behavior of each package is as follows:
+
+* `dotnet-runtime-[major].[minor]`: new patch versions update the package, but new minor or major versions are separate packages.
+* `dotnet-sdk`: `update` rolls forward major, minor, and patch versions.
+* `dotnet-sdk-[major].[minor]`: new patch versions update the package, but new minor or major versions are separate packages.
+* `dotnet-lts`: `update` rolls forward major, minor, and patch versions.
+
+Those are recommendations, and each distro is free to choose what versions to offer, and when. For example, a distro that values stability more than being up to date may choose to release only versions that snap with the LTS release train.
```

Comment:
up to date -> up-to-date

Those are recommendations -> This topic has presented our recommendations for ... however each distro is free to choose what versions to offer and when.